build_tools/github_actions/post_build_upload.py

```diff
@@ -55,6 +56,26 @@ def check_aws_cli_available():
         log("[ERROR] AWS CLI not found in PATH.")
         sys.exit(1)
 
+    if is_windows():
+        get_time_sync_logs()
+
+
+def get_time_sync_logs():
+    startfile = Path("H:\\start.log")
+    timefile = Path("H:\\time.log")
```

Comment:
Document in a comment what the expectations are for this `H` drive. This script should be runnable on multiple machines (developer machines, Azure build VMs, lab test machines, etc.), and this looks specific to our current Azure build VMs.